From 03490ddbd2d46d73ca99039d0c04939748a66b66 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:26:55 +0800
Subject: [PATCH] Revert "USB: android: Use composite class driver for enabling
 and disabling functions"

This reverts commit 65eb4c8d6eecf6f3637efed5be8ab31adf1452ff.
---
 drivers/usb/gadget/android.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/usb/gadget/android.c b/drivers/usb/gadget/android.c
index 72a1aab66214..ce7426be487d 100755
--- a/drivers/usb/gadget/android.c
+++ b/drivers/usb/gadget/android.c
@@ -299,7 +299,6 @@ static struct usb_composite_driver android_usb_driver = {
 	.dev		= &device_desc,
 	.strings	= dev_strings,
 	.bind		= android_bind,
-	.enable_function = android_enable_function,
 };
 
 void android_register_function(struct android_usb_function *f)
-- 
2.35.3

